### ISSUE TITLE
Improve date readibility

### ISSUE THREAD

**Issue by skinny97214** [indieweb/wiki#11]
Currently the events page uses YYYY-MM-DD which has two problems. First, the year is first, so if you scan down the left edge you see only years which is not very useful. Second, there is month/day ambiguity between US and the rest of the world, so I'm not sure whether this is April 9th or September 4th.

Current: 2014-04-09
Desired: 09 April 2014

![screen shot 2014-04-08 at 4 03 41 pm](https://cloud.githubusercontent.com/assets/41135/2650441/9a150a5c-bf72-11e3-98ba-fb8d52a6cf47.png)

**Comment by aaronpk** [comment]
There are actually several articles in several other languages: http://indiewebcamp.com/other-languages

@gregorlove did you ever end up making any more progress on that event template?

**Comment by gRegorLove** [comment]
It's about 90% functional. To get around MediaWiki restrictions on using template variables in `time` elements, it's using value-class pattern and there's some `display: none` being used, unfortunately. I believe it's parsing correctly, though. See http://indiewebcamp.com/Template:one-day-event and http://indiewebcamp.com/User:Gregorlove.com/sandbox

One thing that still needs to be done is wrapping multiple locations in separate `p-location` http://indiewebcamp.com/irc/2015-04-02#t1428004903208

**Comment by gRegorLove** [comment]
I just added support for up to 10 different `p-locations` for an event. I think it should be ready to use, pending feedback. https://indiewebcamp.com/Template:one-day-event

**Comment by tantek** [comment]
IndieWebCamp has both a global community, and a very strong US-representation, so both US-centric Month Day, Year dates, and Europe-centric Day Month Year dates are unacceptable. 

Let's stick with ISO Dates since they work the best globally, and are readable / different enough for US users for them to understand unambiguously as well.

See also:
* https://twitter.com/amazingmap/status/599931666803597312
* https://xkcd.com/1179/

**Comment by skinny97214** [comment]
I agree that if we were using numbers only the ISO format would be preferable,@tantek, but we're not burdened with that constraint. I'm not proposing a numeric date, I'm proposing spelling out the month with letters. The ISO standard wasn't designed with human readability as its primary concern.
